server.js

```diff
@@ -25,12 +52,22 @@ const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
+app.get('/books', (req, res) => {
+  Book.find().then((data) => res.json(data))
+})
+app.get('/books/:ID', async (req, res) => {
+  const { ID } = req.params
+  if (ID) {
+    const response = await Book.find({ bookID: ID })
+    res.json(response)
+  }
+})
 
```

Comment:
Nice endpoints and really good job with your path param to getting the original book id! 💯 
It would have been nice to have som error responses e.g if the user enters an id that does not match or if the server is down for some reason. Otherwise really nice endpoints and working like a charm 🧙